packages/lesswrong/components/tagging/TagPage.tsx

```diff
@@ -983,63 +992,46 @@ const TagPage = () => {
   //   </ContentStyles>
   // );
 
-  const alternatives = (
-    <ContentStyles contentType="tag" className={classes.alternativesContainer}>
-      <div className={classes.alternativesHeader}>
-        {/* Teach me this */}
-        {/* <span className={classes.alternativesTitle}> slower/faster</span> */}
-
+  const linkedTag = (linkedPage: TagBasicInfo) => <div key={linkedPage.slug} className={classes.linkedTag}>
+    <TagsTooltip placement="left" tagSlug={linkedPage.slug}>
+      <Link to={tagGetUrl(linkedPage)}>{linkedPage.name}</Link>
+    </TagsTooltip>
+  </div>
+
+  const hasList = (list: TagBasicInfo[] | undefined) => list && list?.length > 0;
+
+  const { requirements, teaches, lessTechnical, moreTechnical, slower, faster } = selectedLens?.arbitalLinkedPages as ArbitalLinkedPages;
```

Comment:
Why do we need the type coercion?  Also, it's not safe to destructure from something that might be undefined, it'll blow up at runtime.

Reply:
I want to cause the destructured objects to have the right types. Without the coercion, they end up as "any" even if it seems the type should be not that